pom.xml

```diff
@@ -556,7 +556,25 @@
           <workingDirectory>src/main/webapp/WEB-INF/themes/desktop</workingDirectory>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>jib-maven-plugin</artifactId>
+        <version>2.1.0</version>
+        <configuration>
+          <to>
+            <image>docker.io/plos/wombat:${project.version}</image>
+          </to>
+          <from>
+            <image>docker.io/plos/tomcat:latest</image>
```

Comment:
docker complains about this but I assume because we control the image it's OK

Reply:
docker complains because the build is not reproducible when using `latest`

Reply:
That's technically true with any tag (or by having RUN statements in your Docekrfile) but yes I think it's ok in this case. We can version at some point if it's a concern.